src/Files.Uwp/Filesystem/ListedItem.cs

```diff
@@ -133,8 +133,7 @@ public string[] FileTags
 
         public IList<FileTagViewModel> FileTagsUI
         {
-            get => UserSettingsService.PreferencesSettingsService.AreFileTagsEnabled ?
-                FileTagsSettingsService.GetTagsByIds(FileTags) : null;
+            get => FileTagsSettingsService.GetTagsByIds(FileTags);
```

Comment:
Should we still have the null check?

Reply:
Just checked, the property still return 'null' if there are no tags selected.
Functions that use this property check if the value is null